.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,26 @@
 <!--
 Thank you for opening this pull request!
-
-Please make sure that you that you that your pull request aligns with our checklist-- https://github.com/chicken-biryani/geomstats/blob/master/docs/contributing.rst#pull-request-checklist
 -->
 
+## Checklist
+
+- [ ] My pull request has a suggestive title.
+- [ ] If neccessary, my code is [vectorized](https://www.geeksforgeeks.org/vectorization-in-python/).
+- [ ] I have added apropriate unit tests.
+- [ ] I have made sure the code passes all unit tests.
+- [ ] My PR follows PEP8 guidelines (run '`flake8`)
```

Comment:
The link to Pep8

Reply:
Comment about the commands to run can be added?
"$ pip3 install -r dev-requirements.txt
Then you can run the following two commands:

$ flake8 --ignore=D,W503,W504 geomstats examples tests
$ flake8 geomstats/geometry geomstats/learning"